openai/build.gradle.kts

```diff
@@ -80,17 +80,22 @@ kotlin {
     val jvmMain by getting {
       dependencies {
         implementation(libs.logback)
+        api(libs.ktor.client.cio)
```

Comment:
Does the OpenAI module need any ktor deps, or does it get it directly through the client library dependency?

Reply:
It's written in an abstract way, the engines are packaged separately so need to be explicitly imported by a downstream project.

Reply:
thanks @nomisRev !